pubspec.yaml

```
@@ -1,18 +1,17 @@
 name: dart_pact_consumer
 description: A consumer library to build and test pact contracts.
-version: 0.1.0
-# homepage: https://www.example.com
+version: 0.2.0
```

This is a breaking change, consider bumping it to `1.0.0`.
```suggestion
version: 1.0.0
```

---

Semver states modules may break API compability up until version 1.0.0 without bumping majors.
There is a section on the README that states the API might change up until stable version 1.